content/docs/contributing/docs.md

```diff
@@ -109,19 +112,28 @@ If you change source code files, run tests:
 We use [Prettier](https://prettier.io/) to format our source code. Below is a
 set of wrapper commands for your convenience:
 
-- `yarn format-check` - check all source and content files that they are
-  properly formatted. This script does not fix any found issue, only reports
-  them.
-- `yarn format-all` - fix all found problems.
-- `yarn format-staged` - same, but only on staged files.
-- `yarn format <file>` - run this script `yarn format <file-name>` to format a
-  specific file.
+- `yarn format-all` - Run `prettier --write` on all files to find formatting
+  issues and automatically fix as many as possible.
+- `yarn format-check-all` - Run `prettier --check` on all files, exiting with an
+  error if any issues are found. This is particularly useful for CI.
+- `yarn format-staged` - alias for `lint-staged`, which will run all applicable
+  linters on all files staged in your copy of the git repository.
+- `yarn format <file>` - Run `prettier` on a specific file.
 
 We use linters (e.g. [ESLint](https://eslint.org/)) to check source code style
 and detect different errors:
 
-- `yarn lint-ts` - lint source code files (`.ts`, `.js`, `tsx`, etc).
-- `yarn lint-css` - lint `.css` files.
+- `yarn lint` - run `eslint` on all compatible source code files: `js`, `jsx`,
+  `ts`, `tsx`, `json`.
+- `yarn lint-fix` - use `eslint --fix` to automatically fix as many issues as
+  possible.
+- `yarn lint-ts` - uses `tsc` to attempt to compile the project, ensuring all
+  TypeScript types are properly formatted.
+- `yarn lint-css` - lint `.css` files with `stylelint`, which covers more than
+  `prettier` does alone.
+
+To use the `fix` variant of all of these tools in one command, use
+`yarn fix-all`.
```

Comment:
Rename some commands to what I hope is a more understandable set that correlates with the tools used.

`yarn fix-all` is added for the "I want to run and fix everything possible or error when it can't" command, to replace the convenience of having `eslint` and `tsc` combined on `lint-ts`

Open to different names if there are other opinions.